app/src/marketplace/types/project-types.ts

```diff
@@ -13,6 +14,16 @@ export type PendingChanges = {
     requiresWhitelist?: boolean
     pricingTokenAddress?: Address
 }
+export type SalePoint = {
+    chain: string,
+    beneficiaryAddress: Address
+    pricePerSecond: NumberString
+    priceCurrency: ContractCurrency
+    timeUnit: TimeUnit | null | undefined
+    price: NumberString
```

Comment:
These 3 we don't need anymore. `PricePerSecond` and `PricingTokenAddress` should be all we need. We can calculate that `timeUnit` thing from `PricePerSecond` to show user meaningful string like 300 DATA per hour.